docs/items/index.md

```diff
@@ -1,72 +1,127 @@
-Items
-=====
+# Items
 
-Along with blocks, items are a key component of most mods. While blocks make up the level around you, items exist within inventories.
+Along with blocks, items are a key component of Minecraft. While blocks make up the world around you, items exist within inventories.
 
-Creating an Item
-----------------
+## What Even Is an Item?
 
-### Basic Items
+Before we get further into creating items, it is important to understand what an item actually is, and what distinguishes it from, say, a [block][block]. Let's illustrate this using an example:
 
-Basic items that need no special functionality (think sticks or sugar) do not need custom classes. You can create an item by instantiating the `Item` class with an `Item$Properties` object. This `Item$Properties` object can be made via the constructor and customized by calling its methods. For instance:
+- In the world, you encounter a dirt block and want to mine it. This is a **block**, because it is placed in the world. (Actually, it is not a block, but a blockstate. See the [Blockstates article][blockstates] for more detailed information.)
+- Once you have mined the block, it is removed (= replaced with an air block) and the dirt drops. The dropped dirt is an item **entity**. This means that like other entities (pigs, zombies, arrows, etc.), it can inherently be moved by things like water pushing on it, or burned by fire and lava.
+- Once you pick up the dirt item entity, it becomes an **item stack** in your inventory. An item stack is, simply put, an instance of an item with some extra information, such as the stack size.
+- Item stacks are backed by their corresponding **item** (which is what we're creating). Items hold information that is the same across all items (for example, every iron sword has a max durability of 250), while item stacks hold information that can be different between two similar items (for example, one iron sword has 100 uses left, while another iron sword has 200 uses left).
```

Comment:
It might be best to equate this to an item stack is backed by an item the same way a blockstate is backed by a block. This is not 100% true, but it could be used as a rough comparison.